v2/php/examples/GenerateSavedReport.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Retrieves a report for the specified ad client.
+ *
+ * Tags: accounts.reports.saved.generate
+ */
+class GenerateSavedReport {
+  /**
+   * Retrieves a report for the specified ad client.
+   *
+   * @param $service Google_Service_AdSense AdSense service object on which to
+   *     run the requests.
+   * @param $savedReportId string the saved report ID on which to run the
+   *     report.
+   */
+  public static function run($service, $savedReportId) {
```

Comment:
I believe this requires account ID.
```suggestion
  public static function run($service, $savedReportId, $accountId) {
```

Reply:
Actually the accountId isn't needed, and it was just a leftover print statement. PHP is really lenient about missing variables, so it just printed an empty string. Anyway, I've removed it in GenerateSavedReport.php.